frontend/src/components/common/buttons/Button.tsx

```diff
@@ -188,6 +188,7 @@ const StyledButton = styled(BootstrapButton)`
       line-height: 3rem;
       justify-content: left;
       letter-spacing: unset;
+      text-align: left;
```

Comment:
this aligns the text inside a button of variant=link to the left so it works with long text. I checked the places where we use this button variant and they looked fine. I will check with sue to make sure nothing is looking messy after this.